main.c~

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
Not sure why these .c~ files got included at all. Have I configured something in the Makefile wrong?